jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/APIKeys/RotateAPIKeysRule.java

```diff
@@ -95,14 +99,20 @@ private boolean rotateApiKeys(String vmEsURL, Map<String, Object> mustFilter) th
         if (hitsJsonArray.size() > 0) {
             logger.info("hit array size {}", hitsJsonArray.size());
             JsonObject apiKeys = (JsonObject) ((JsonObject) hitsJsonArray.get(0)).get(PacmanRuleConstants.SOURCE);
-            String cdDate=apiKeys.get("createdDate").getAsString();
-            SimpleDateFormat sdf=new SimpleDateFormat("EEE MMM dd HH:mm:ss yyyy");
-            TimeZone timeZone=TimeZone.getTimeZone("IST");
-            sdf.setTimeZone(timeZone);
-            cdDate=cdDate.replace("GMT ", "");
-            Date date=  sdf.parse(cdDate);
-           if (TimeUnit.DAYS.convert(new Date().getTime()-date.getTime(), TimeUnit.MILLISECONDS)<=90){
-                    validationResult= true;
+            String createdDateStr = apiKeys.get("createdDate").getAsString();
+            String istZoneIdStr = "Asia/Kolkata";
```

Comment:
wait a second... why IST zone is here? It everywhere... this must go for sure. UTC I guess is what used in AWS across whole cloud.

Reply:
Addressed in latest push. Recent code does not do IST conversion